spec-main/fixtures/extensions/chrome-runtime/main.js

```diff
@@ -1,6 +1,40 @@
 /* eslint-disable */
-document.documentElement.textContent = JSON.stringify({
-  manifest: chrome.runtime.getManifest(),
-  id: chrome.runtime.id,
-  url: chrome.runtime.getURL('main.js')
+
```

Comment:
I converted this script to another pattern we use such that each API can be tested in isolation.

Reply:
Nice change 👍